projects/igniteui-angular/src/lib/directives/mask/mask-parsing.service.ts

```diff
@@ -1,45 +1,48 @@
+import { Injectable } from '@angular/core';
+
 /**
  * @hidden
  */
-export const MASK_FLAGS = [ 'C', '&', 'a', 'A', '?', 'L', '9', '0', '#' ];
+export const MASK_FLAGS = ['C', '&', 'a', 'A', '?', 'L', '9', '0', '#'];
 
 /**
  * @hidden
  */
 export const KEYS = {
-    Ctrl : 17,
-    Z : 90,
-    Y : 89,
-    X : 88,
-    BACKSPACE : 8,
-    DELETE : 46
-  };
-
+    Ctrl: 17,
+    Z: 90,
+    Y: 89,
+    X: 88,
+    BACKSPACE: 8,
+    DELETE: 46
+};
```

Comment:
Can you merge these with [those](https://github.com/IgniteUI/igniteui-angular/blob/9c70ca8c48510bc0c113d297ef8aa31c36991a25/projects/igniteui-angular/src/lib/core/utils.ts#L133) an use them instead of redeclaring similar enumerations in the lib?